code/game/jobs/job/science.dm

```diff
@@ -65,7 +65,7 @@
 
 	uniform = /obj/item/clothing/under/rank/scientist
 	suit = /obj/item/clothing/suit/storage/toggle/labcoat/science
-	shoes = /obj/item/clothing/shoes/white
+	shoes = /obj/item/clothing/shoes/medical

```

Comment:
You are reusing the medical shoes in science.
Create a stand alone type for that or change the path.

Reply:
@_@ they're all the same chem resistant shoes even if in the code they're tagged does Medical. They don't even have the word Medical in their name in Game. You want me to create four sets of exactly identical in appearance white shoes that all do something slightly different for each department?

Reply:
As I mentioned in my review:

> Create a shoe type for the scientists that has spill resistance
> Create a shoe type for the doctors that has bio resistance (bio armor)
> Create a shoe type for the viro/chem that has bio and spill resistance